src/components/AccountLoginForm.jsx

```diff
@@ -5,7 +5,7 @@ import classNames from 'classnames'
 import statefulForm from '../lib/statefulForm'
 import Field, { PasswordField, DropdownField, CheckboxField } from './Field'
 
-const AccountLoginForm = ({ t, konnector, customView, fields, error, dirty, submitting, values, submit, onCancel }) => {
+const AccountLoginForm = ({ t, konnector, customView, fields, error, dirty, submitting, deleting, values, submit, onDelete, onCancel }) => {
   const isUpdate = !!values.login || !!values.access_token
```

Comment:
Not wrong, we also considerate the deletion as an update as well, we can keep it so.